### ISSUE TITLE
Session Time out

### ISSUE THREAD

**Issue by matt-dobson** [OHS-Hosting-Infrastructure/complaint-tracker#61]
https://gsa-tts.slack.com/archives/C01TT2YNX0R/p1628862775002100

**Comment by jduss4** [comment]
Response from Alana:
https://gsa-tts.slack.com/archives/C026KUV3JBU/p1628865968005000

**Comment by matt-dobson** [comment]
Slack thread: https://gsa-tts.slack.com/archives/C01TT2YNX0R/p1628862775002100

**Comment by matt-dobson** [comment]
needs time - 2hrs?

**Comment by matt-dobson** [comment]
Needs an actual API response to exist.

**Comment by matt-dobson** [comment]
Accessibility requires the ability to extend a session, so we need to track if the time of someone’s session and then give them the option to extend (probably by hitting HSES to extend their session? not sure.)

https://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits-required-behaviors.html

**Comment by matt-dobson** [comment]
This still needs details on session time, countdown time, and extension time.
